js/news.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import * as constants from './constants.js';
```

Comment:
Just FYI other choice:
```javascript
import { getMoreButton, apiVersion, apiKey, articlesWrapper, getMoreButton } from './constants.js';

Reply:
it's more comfortable for me, but I keep in mind your remarks